### PR TITLE
add `allowed_on_disk_full` field for message Context

### LINKED ISSUE #784: add `allowed_on_disk_full` field for message Context

### PR DESCRIPTION
**Pull request by tier-cap**
### What problem does this PR solve?

Issue Number：close #784

###  What is changed and how it works?

1. add a flag used by tidb to tell tikv which operation can be allowed when tikv disk full happens.

### Check List

not available

### Release note

not available

### CODE REVIEW

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -683,6 +683,9 @@ message Context {
 
     // Any additional serialized information about the request.
     bytes resource_group_tag = 18;
+
+	// Allowed when tikv disk full happened.
+	bool allowed_on_disk_full = 19;
```
Comment (hicqu) [inline proto/kvrpcpb.proto:688]: Please fix the wrong indent.

**Comment by hicqu** [comment]
LGTM.

**Review by hicqu — APPROVED** [review]

**Review by NingLin-P — APPROVED** [review]

File: proto/kvrpcpb.proto [hunk 2]
```diff
@@ -685,6 +685,9 @@ message Context {
 
     // Any additional serialized information about the request.
     bytes resource_group_tag = 18;
+
+    // Let cur command execute be allowed when tikv disk full happened.
```
Comment (hicqu) [inline proto/kvrpcpb.proto:689]: Not only CUR commands. Raw commands can only use this field if necessary.

**Review by tisonkun — APPROVED** [review]

**Pull request merged**